doc/source/dem_class.md

```diff
@@ -81,9 +89,12 @@ import geoutils as gu
 fn_glacier_outlines = xdem.examples.get_path("longyearbyen_glacier_outlines")
 vect_gla = gu.Vector(fn_glacier_outlines)
 
+# Crop outlines to those intersecting the DEM
+vect_gla = vect_gla.crop(dem)
+
 # Plot the DEM and the vector file
 dem.plot(cmap="terrain", cbar_title="Elevation (m)")
-vect_gla.plot(dem)  # We pass the DEM as reference for the plot CRS/extent
+vect_gla.plot(dem, ec="k", fc="none")  # We pass the DEM as reference for the plot CRS/extent
```

Comment:
Actually, on the figure, the plot extent is still that of the outlines, not the DEM...

Reply:
Corrected, the reference is only for the CRS